recipes/mlconjug/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "mlconjug" %}
+{% set version = "3.1.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a0ba92de0579ed0e74fc0d594bff3d4895728bd6e21b881f5c2b084c328cefbc" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
+  preserve_egg_dir: True
+  number: 0
+  entry_points:
+    - mlconjug=mlconjug.cli:main
+  script: $PYTHON -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+    - pip
+  host:
```

Comment:
I think that the `host` section should just include `python`, `pip`, `setuptools` and `pytest-runner`. Everything else appears to be a `run` dependency, but not required for building the package.